lib/commands/description.js

```diff
@@ -110,8 +123,12 @@ async function main() {
         // Update plugin description
         info.description = description;
 
+        const savePluginOptions = {
```

Comment:
Since this `const`ant is only used once, we should probably inline its value.

Reply:
Adjusted in <https://github.com/VIISON/scs-commander/pull/23/commits/96294e6e6e12bafbe0b6a696fd3d63f88868fecf>.